LiskHQ/lisk-hub
# Update account initialization banner to the new design - closes #2257

### What issue have I solved?

-- #2257 

### How have I implemented/fixed it?

The PR contains a new feature element that has a few details/component. Those component are:
- icon
- text to display/show
- link that will redirect to an external page (lisk documentation)
- button to init the account (on click will redirect to the send component to init the account)

This component looks like the design and it is intended to be only display when user create an account for first time and there is no transactions at all.

### How has this been tested?

The tests that I made to check this is working properly are the follow:

- login with an account already init, then the banner should NOT be visible.
- login with an uninitialize account, then the banner should be visible in all pages after login, if logout banner will NOT be display.
- login with an uninitialize account, then do click in the Initialize account button in the banner and send the first transaction to init the account after this the banner should disappear. After this logout and login with the same account and the banner should NOT show up at all.

### Review checklist
- The PR follows our [Test guide](/LiskHQ/lisk-hub/blob/development/docs/TEST_GUIDE.md)
- The PR follows our [CSS guide](/LiskHQ/lisk-hub/blob/development/docs/CSS_GUIDE.md)
